crawl-ref/source/dat/descript/zh/items.txt

```diff
@@ -261,71 +246,70 @@ book of spatial translocations
 %%%%
 book of spectacle
 
-一本覆盖着闪烁的亮片的书。每一页都有一阵即兴的灯光秀，
-显然这本书只留给了最俗气的巫师。
+这本上满是闪烁的亮片。每翻动一页就有一阵即兴的灯光秀，
+显然这本书是最俗气的巫师留下的。
 %%%%
 book of the spheres
 
-这本书包含与球体相关的综合、全面的咒语。
-它的最后几页描写了一个名为“佐特宝珠”的咒语，但遗憾的是，
-这本书的剩余部分已经完全蒸发了。
+这本书中包含着与球体相关的综合而全面的咒语。
+它的最后几页写着一个名为“佐特宝珠”的咒语，
+但遗憾的是，这本书的剩余部分已经完全蒸发了。
 %%%%
 book of storms
 
-魔法师兰石写的一本书，他专注于于研究和控制自然现象。
+由魔法师兰石编写的一本书，他专注于于研究和控制自然现象。
 石的大部分作品已经失传，但仍有些被幸运（或不道德）的巫师收藏着，
 因此能够从中获得一份副本。
 %%%%
 book of the dragon
 
-一本关于控制龙和其他关于龙的咒语魔法书。
+一本写有控制龙和其他关于龙的咒语的魔法书。
 %%%%
 book of the wilderness
 
-一本从遥远的野性的土地上撕扯来的书，记载着传说。
+这本记载着传说的书是从遥远的蛮荒土地上撕扯过来的，
 可以把那里的生物召唤来为读者服务。
 {{
     if you.can_smell() then
-        return "闻起来有股霉味，令人难受。"
+        return "它有股难闻的霉味。"
     end
 }}
 %%%%
 book of the earth
 
-一本从大地深处汲取了巨大力量的指南。
-原本是由丘陵矮人和山地矮人合著，
-但由于一波巨大的地震使他们的文明陷入毁灭，编纂时间被缩短了。
+这本指南从大地深处汲取了巨大的力量。它原本由丘陵矮人和山地矮人合著，
+但由于一波巨大的地震毁灭了他们的文明，编纂时间被缩短了。
 %%%%
 book of the hunter
 
-一本关于狩猎的平淡无奇的书。有人翻阅完这本书，
-所做的修改和严厉的批评，涂满了整本书，
-还划掉了整个章节，换用了能完成相同任务的咒语。
+一本关于狩猎的平淡无奇的书。
+有人翻阅完这本书，所做的修正和严厉的批评涂满了整本书，
+甚至还划掉了整个章节，换用了能完成相同任务的咒语。
 %%%%
 book of the moon
 
-由一位疯狂的巫师所写的书，
-他确信天边的月亮不是由闪亮的光构成的，而是像脚下那样坚硬的岩石构成的。
-许多书页已被撕得面目全非，
-似乎是被一对有力的爪子撕碎的……但巨怪拿着一本魔法书做什么？
+这本书由一位疯狂的巫师所写，他深信天边的月亮不是由闪亮的光构成的，
+而是像脚下那样坚硬的岩石构成的。
+许多书页已被撕得面目全非，这似乎是被一对有力的爪子撕碎的……
+但巨魔拿着一本魔法书做什么？
 %%%%
 book of the senses
 
-一部学术作品，描述了生物用来感知周围世界的器官。
-其中包含了一些咒语，以演示所描述的原理。
+这部学术作品，描述着生物用来感知周围世界的器官。
+它包含了一些咒语，以演示其所描述的原理。
 %%%%
 book of the sky
 
-一本包含了强大的气元素咒语的魔法书。
-伟大的弥诺陶洛斯英雄乌克斯胡恩困在一个有着特别任性的幽默感的巫师塔塔顶时，
-得到了这本书的副本，他用这本书的书页建造了一个简陋的滑翔机，并飞向自由。
+这本魔法书中包含着强大的气系魔法咒语。
+伟大的牛头人英雄乌克斯胡恩被困在一个有着特别过分的幽默感的巫师塔塔顶时，
+得到了这本书的副本，他用这本书的书页搭建了一个简陋的滑翔机，飞向了自由。
```

Comment:
I'm very curious - why did you change this one? Google Translate suggests it went from "a particularly wayward sense of humor" to " a particularly over-the-top sense of humor"... the first one seems closer to the original "a perverse sense of humor" to me, though really neither comes that close. (The wizard thought it would be funny to give a minotaur a book of magic spells, since of course minotaurs can't cast spells - perverse!)

I don't know Chinese, though, and Google Translate could be misleading me. So, I'm just curious.